static/js/public/expandable-details.js

```diff
@@ -1,21 +1,21 @@
 export default function initExpandableSnapDetails() {
-  const moreEl = document.querySelector(".snap-description-more");
-  const textEl = document.querySelector(".js-snap-description-text");
-  const detailsEl = document.querySelector(".js-snap-description-details");
+  const showMoreContainer = [].slice.call(
+    document.querySelectorAll("[data-js='js-show-more']")
+  );
 
-  if (textEl.clientHeight > detailsEl.clientHeight) {
-    textEl.parentNode.classList.add("snap-description-collapsed");
-    moreEl.classList.remove("u-hide");
-  }
+  if (showMoreContainer) {
```

Comment:
This will be defined even if there are no elements, probably want to test `showMoreContainer.length > 0`